modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/util/ExportImportTaskUtil.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.batch.engine.util;
+
+import com.liferay.headless.batch.engine.client.dto.v1_0.ExportTask;
+import com.liferay.headless.batch.engine.client.dto.v1_0.ImportTask;
+import com.liferay.headless.batch.engine.client.serdes.v1_0.ExportTaskSerDes;
+import com.liferay.headless.batch.engine.client.serdes.v1_0.ImportTaskSerDes;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.test.util.HTTPTestUtil;
+import com.liferay.portal.kernel.util.Http;
+import com.liferay.portal.kernel.util.MapUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.Map;
+
+import org.junit.Assert;
+
+/**
+ * @author Alberto Javier Moreno Lage
+ */
+public class ExportImportTaskUtil {
+
+	public static ExportTask postExportTask(
+			String className, String expectedExecuteStatus,
+			Map<String, String> parameters)
+		throws Exception {
+
+		ExportTask exportTask = ExportTaskSerDes.toDTO(
+			HTTPTestUtil.invokeToString(
+				null,
+				StringBundler.concat(
+					"headless-batch-engine/v1.0/export-task/", className,
+					"/JSON", _getQueryString(parameters)),
+				Http.Method.POST));
+
+		String externalReferenceCode = exportTask.getExternalReferenceCode();
+
+		while (true) {
+			exportTask = ExportTaskSerDes.toDTO(
+				HTTPTestUtil.invokeToString(
+					null,
+					"headless-batch-engine/v1.0/export-task/by-external-" +
+						"reference-code/" + externalReferenceCode,
+					Http.Method.GET));
+
+			if (!StringUtil.equals(
+					exportTask.getExecuteStatusAsString(), "STARTED")) {
```

Comment:
I think the INITIAL status is also possible. Should we replace with COMPLETED / FAILED instead?

Reply:
Hmm... but that would only work if the expectedExecuteStatus is COMPLETED or FAILED. That may not always be the case.

Reply:
but are we expecting to have a different expected status from COMPLETED or FAILED? there is only one status more, INITIAL, and there are no tests now that are expecting that status